### ISSUE TITLE
Protobuf files need to be regenerated for protoc versions > 3.19

### ISSUE THREAD

**Issue by AndrewFasano** [panda-re/panda#1322]
It seems like our generated pandalog, protobuf files are old enough that they need to be regenerated to make modern protobuf happy. I have protobuf version 4.23.4 installed and get the following error whenever I try running pypanda.

Easy fix is to regenerate these files, but I wonder if we might want to make the import of the plog infrastructure optional or just allow it to fail. It's not often used with pypanda scripts and probably shouldn't be fatal if it's not set up right.

```
Traceback (most recent call last):
  File "run.py", line 18, in <module>
    from pandare import Panda, blocking
  File "/home/andrew/git/panda/panda/python/core/pandare/__init__.py", line 14, in <module>
    from .plog_reader import PLogReader
  File "/home/andrew/git/panda/panda/python/core/pandare/plog_reader.py", line 8, in <module>
    import pandare.plog_pb2
  File "/home/andrew/git/panda/panda/python/core/pandare/plog_pb2.py", line 36, in <module>
    _descriptor.FieldDescriptor(
  File "/home/andrew/git/panda/panda/python/venv/lib/python3.10/site-packages/google/protobuf/descriptor.py", line 561, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

**Comment by AndrewFasano** [comment]
Turns out these files aren't even tracked in our repo 🤦 they're just built as we build panda. panda/panda/Makefile.panda.target specifies how they're built with `protoc` and if the host protoc doesn't match the host's python protobuf version, you'll run into the issue I encountered.

Given that most pypanda analyses don't use protobuf, a better solution might be to make the PLogReader import optional

**Comment by AndrewFasano** [comment]
Fixed after 0d98fa49b723a1f623b18de1bbd27d025a208200
